src/examples/forms/multithumbrange/MultiThumbRangeDefault.tsx

```diff
@@ -0,0 +1,35 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React, { useState } from 'react';
+import { Row, Col } from '@zendeskgarden/react-grid';
+import { Field, Label, MultiThumbRange } from '@zendeskgarden/react-forms';
+
+const Example = () => {
+  const [minVal, setMinVal] = useState<number | undefined>();
+  const [maxVal, setMaxVal] = useState<number | undefined>();
```

Comment:
Since the default value of these states is `undefined`, the generic can be updated to `useState<number>();` to take advantage of type inference. TypeScript will infer that the default value is `undefined` so it doesn't need to be explicitly passed.